src/doc/src/reference/build-scripts.md

```diff
@@ -130,6 +130,8 @@ one detailed below.
   compiler.
 * [`cargo::rustc-cfg=KEY[="VALUE"]`](#rustc-cfg) --- Enables compile-time `cfg`
   settings.
+* [`cargo::rustc-check-cfg=CHECK_CFG`](#rustc-check-cfg) -- Enables compile-time
```

Comment:
This doesn't enable compile-time checking but registers `cfg`s as valid for use with the compile-time checking

Reply:
Changed to `Register custom cfgs as expected for compile-time checking of configs`